util/src/main/i18n/templates/ui.properties

```diff
@@ -43,6 +43,54 @@ setting.visibility.hidden = Hidden
 
 setting.observersOnly = Sorry, this command is only for observers.
 
+setting.chat = Chat
+setting.chat.lore = Channel: {0}
+setting.chat.value.team = Team
+setting.chat.value.global = Global
+setting.chat.value.admin = Admin
+
+setting.death = Death messages
+setting.death.lore = Status: {0}
+setting.death.value.own = Own
+setting.death.value.all = All
+
+setting.picker = Team picker
+setting.picker.lore = Status: {0}
+setting.picker.value.auto = Auto
+setting.picker.value.on = On
+setting.picker.value.off = Off
+
+setting.join = Join messages
+setting.join.lore = Status: {0}
+setting.join.value.on = All
+setting.join.value.off = None
+
+setting.message = Direct messages
+setting.message.lore = Status: {0}
+setting.message.value.on = All
+setting.message.value.off = None
+
+setting.observers = Observer visibility
+setting.observers.lore = Status: {0}
+setting.observers.value.on = Shown
+setting.observers.value.off = Hidden
+
+setting.sounds = Sounds
+setting.sounds.lore = Status: {0}
+setting.sounds.value.all = All
+setting.sounds.value.messages = DMs
+setting.sounds.value.none = None
+
+setting.vote = Vote menu
+setting.vote.lore = Status: {0}
+setting.vote.value.on = On
+setting.vote.value.off = Off
+
+setting.stats = Stats
+setting.stats.lore = Tracking: {0}
+setting.stats.value.on = On
+setting.stats.value.off = Off
+
```

Comment:
Duplicate strings should be merged

All strings with "off" and "on" should be replaced with `misc.on` and `misc.off`

`setting.stats` should be removed in favour of `match.stats.overall` (might want to change the key name)

Strings for "Shown and Hidden" already exists (might want to change key name maybe)